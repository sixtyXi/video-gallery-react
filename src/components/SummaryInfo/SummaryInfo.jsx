// @flow

import * as React from 'react';

import styles from './SummaryInfo.scss';
import { ContentWrapper } from '../ContentWrapper/ContentWrapper';

export const SummaryInfo = ({ children }: { children: React.Node | null }) => {
  return (
    <div className={styles.summaryWrapper}>
      <ContentWrapper>{children}</ContentWrapper>
    </div>
  );
};

SummaryInfo.defaultProps = {
  children: null
};
