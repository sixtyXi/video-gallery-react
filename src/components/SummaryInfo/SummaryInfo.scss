@import '../../shared/_typography.scss';

.summaryWrapper {
  width: 100%;
  padding: 5px 0;
  background: lightgray;
  font-weight: bold;
  font-size: $size-s;

  & > section {
    display: flex;
    justify-content: space-between;
    max-width: 700px;
    white-space: nowrap;
  }
}
