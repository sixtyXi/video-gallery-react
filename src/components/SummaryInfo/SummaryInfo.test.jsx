import React from 'react';
import { shallow } from 'enzyme';

import { SummaryInfo } from './SummaryInfo';

describe('SummaryInfo', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<SummaryInfo />);

    expect(wrapper).toMatchSnapshot();
  });
});
