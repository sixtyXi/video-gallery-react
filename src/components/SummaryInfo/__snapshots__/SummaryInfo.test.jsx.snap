// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SummaryInfo renders correctly 1`] = `
<div
  className="summaryWrapper"
>
  <ContentWrapper />
</div>
`;
